web/frontend/src/components/arm.vue

```diff
@@ -25,7 +25,7 @@ export interface RawArmStatus extends ArmStatus {
   end_position: Record<string, number>
 }
 
-type Field = 'x' | 'y' | 'z' | 'ox' | 'oy' | 'oz' | 'theta'
+type Field = 'x' | 'y' | 'z' | 'oX' | 'oY' | 'oZ' | 'theta'
```

Comment:
i opted to just use the field names that we get when calling `proto.toObject()` rather than overwrapping... even though it's probably worse in this case. see https://github.com/viamrobotics/viam-typescript-sdk/pull/55 for details